conans/client/source.py

```diff
@@ -26,29 +25,51 @@ def remove_source(raise_error=False):
             if raise_error or isinstance(e_rm, KeyboardInterrupt):
                 raise ConanException("Unable to remove source folder")
 
-    if force:
-        output.warn("Forced removal of source folder")
-        remove_source(raise_error=True)
-    elif os.path.exists(dirty):
-        output.warn("Trying to remove dirty source folder")
-        remove_source(raise_error=True)
-    elif conan_file.build_policy_always:
-        output.warn("Detected build_policy 'always', trying to remove source folder")
-        remove_source(raise_error=True)
+    def create_source_dirty_flag():
```

Comment:
Yes, I've added this when copying save(dirty, "") and os.remove(dirty) lines so that accompanying comments ("# Creation of DIRTY flag" and "# Everything went well, remove DIRTY flag") would be not needed anymore - because function name will be self explanatory. Would you like me to revert this?